Missing "computation_time.py" file at tesseroid-variable-density/code/scripts/

When I type `make all` at tesseroid-variable-density-master/code, it get errors like:


python2.7 scripts/computation_time.py;  python2.7 scripts/density-based-discretization.py;  python2.7 scripts/exponential_density.py;  python2.7 scripts/grid_search.py;  python2.7 scripts/linear_density.py;  python2.7 scripts/neuquen_basin.py;  python2.7 scripts/sine_density.py;
python2.7: can't open file 'scripts/computation_time.py': [Errno 2] No such file or directory
Traceback (most recent call last):
  File "scripts/density-based-discretization.py", line 8, in <module>
    from tesseroid_density import tesseroid
ImportError: No module named tesseroid_density
Traceback (most recent call last):
  File "scripts/exponential_density.py", line 11, in <module>
    from tesseroid_density import tesseroid
ImportError: No module named tesseroid_density
Traceback (most recent call last):
  File "scripts/grid_search.py", line 11, in <module>
    from tesseroid_density import tesseroid
ImportError: No module named tesseroid_density
Traceback (most recent call last):
  File "scripts/linear_density.py", line 11, in <module>
    from tesseroid_density import tesseroid
ImportError: No module named tesseroid_density
Traceback (most recent call last):
  File "scripts/neuquen_basin.py", line 11, in <module>
    from tesseroid_density import tesseroid
ImportError: No module named tesseroid_density
Traceback (most recent call last):
  File "scripts/sine_density.py", line 11, in <module>
    from tesseroid_density import tesseroid
ImportError: No module named tesseroid_density
make: *** [figures] Error 1

"No module named tesseroid_density" means the Issues #73 Missing "Makefile" file at tesseroid-variable-density/code/tesseroid_density/

Because it misses the `Makefile` at tesseroid-variable-density/code/tesseroid_density/, it cann't build the module "tesseroid_density".

--- a/code/Makefile
+++ b/code/Makefile
@@ -10,8 +10,7 @@ PACKAGE = tesseroid_density
 SCRIPTSDIR = scripts
 RESDIR = $(SCRIPTSDIR)/results
 
-SCRIPTS = $(SCRIPTSDIR)/computation_time.py             \
-          $(SCRIPTSDIR)/density-based-discretization.py \
+SCRIPTS = $(SCRIPTSDIR)/density-based-discretization.py \
           $(SCRIPTSDIR)/exponential_density.py          \
           $(SCRIPTSDIR)/grid_search.py                  \
           $(SCRIPTSDIR)/linear_density.py               \

